nncf/torch/model_transformer.py

```diff
@@ -0,0 +1,97 @@
+                insertion_transformations.append(transformation)
+            if isinstance(transformation, PTModelExtractionCommand):
+                extraction_transformations.append(transformation)
+            if isinstance(transformation, PTBiasCorrectionCommand):
+                bias_correction_transformations.append(transformation)
+
+        if extraction_transformations:
+            return self._apply_extraction_transformations(extraction_transformations)
+
+        if insertion_transformations:
+            self._apply_output_insertion_transformations(insertion_transformations)
+        if bias_correction_transformations:
+            self._apply_bias_correction_transformations(bias_correction_transformations)
+        return self._model
+
+    def _apply_output_insertion_transformations(self, transformations: List[TransformationCommand]) -> None:
+        fns_grouped_by_points = {}  # type: Dict[PTInsertionPoint, List[Tuple[Callable, TransformationPriority]]]
+        for transformation_command in transformations:  # type: PTInsertionCommand
+            target_point = transformation_command.target_point  # type: PTTargetPoint
+            target_node_name = target_point.target_node_name
+            pt_ip = PTInsertionPoint(
+                target_type=target_point.target_type,
+                op_address=self._node_to_op_address_mapping[target_node_name],
+                input_port_id=target_point.input_port_id,
+            )
+            fn = transformation_command.fn
+            if target_point.type is TargetType.OPERATION_WITH_WEIGHTS:
+                fn = UpdateWeight(fn)
+            tup = (fn, transformation_command.priority)
+            if pt_ip not in fns_grouped_by_points:
+                fns_grouped_by_points[pt_ip] = [tup]
+            else:
+                fns_grouped_by_points[pt_ip].append(tup)
+
+        for pt_ip, fn_list_with_priority in fns_grouped_by_points.items():
+            fn_list_with_priority = sorted(fn_list_with_priority, key=lambda x: x[1])
+            self._model.insert_at_point(pt_ip, [x[0] for x in fn_list_with_priority])
+
+    def _apply_extraction_transformations(self, transformations: List[TransformationCommand]) -> nn.Module:
+        assert len(transformations) == 1, "Support only one node extraction"
+        extracted_module = self._model.get_containing_module(transformations[0].node_name)
+        # Return copy of the module without pre and post operations.
+        extracted_module = copy.deepcopy(extracted_module)
+        extracted_module.reset()
+        return extracted_module
+
+    def _apply_bias_correction_transformations(self, transformations: List[TransformationCommand]) -> None:
+        for transformation in transformations:
+            node_name = transformation.target_point.target_node_name
+            node = self._model.get_containing_module(node_name)
+            node.bias.data = transformation.bias_value
```

Comment:
We've agreed to avoid in-place transformations, didn't we?

Reply:
Not sure. Why not, and what alternative? Create new Torch model and change bias in new model? 
@vshampor

Reply:
PTModelTransformation does it in-place. https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/nncf_network.py#L948
It can be change but i prefer do it in another PR. 
@vshampor what you think about it?